src/commands/test_scenario_1.rs

```diff
@@ -0,0 +1,162 @@
+//! `test` - Scenario 1: Three parties,
```

Comment:
Let's add some explanation what's going on, who are these parties. Also maybe a little more descriptive filename?

Reply:
Changed the name and added a description of the scenario.